xilem/src/app_main.rs

```diff
@@ -58,8 +58,8 @@ impl<T: Send + 'static, V: View<T> + 'static> AppLauncher<T, V> {
             QUIT_MENU_ID,
             "E&xit",
             Some(&HotKey::new(SysMods::Cmd, "q")),
+            Some(false),
             true,
-            false,
```

Comment:
Btw, the order of arguments `enabled` and `selected` has changed in druid-shell ([commit](https://github.com/linebender/druid/commit/7c08b326578aa2b25057f5f1af09f9a6b1f51da4#diff-0f63e54d7096d613a02825d47679656d3cd119abea8bc85c59cba648269b6ae0)), so I swapped true & false here too. But the upstream [xilem](https://github.com/linebender/xilem/blob/51ae4d2bfc9d9bcf9e16f7f90a2f85860022cb99/src/app_main.rs#L71) has it in the old order, while [glazier](https://github.com/linebender/glazier/blob/bb4bc453a949b74fc4de16ce6be66b901dcf4ab4/src/menu.rs#L69) matches the latest druid-shell. So I guess it's a regression in Xilem and the quit menu should be unchecked but enabled, is that right? Happy to make a patch to Xilem.

Reply:
Yes this should be fixed in Xilem proper too. A PR would be wonderful.

Reply:
on it

Reply:
[Done](https://github.com/linebender/xilem/pull/42)